Extensions/Effects/JsExtension.js

```diff
@@ -520,6 +552,113 @@ module.exports = {
         .setType('boolean')
     );
 
+    const glitchEffect = extension
+      .addEffect('Glitch')
+      .setFullName(_('Glitch'))
+      .setDescription(_('Applies a glitch effect to an object.'))
+      .addIncludeFile('Extensions/Effects/pixi-filters/filter-glitch.js')
+      .addIncludeFile('Extensions/Effects/glitch-pixi-filter.js');
+    const glitchProperties = glitchEffect.getProperties();
+    glitchProperties.set(
+      'slices',
+      new gd.PropertyDescriptor(/* defaultValue= */ '5')
+        .setLabel(_('Slices (between 2 and 20)'))
+        .setType('number')
+        .setDescription('The maximum number of slices')
+    );
+    glitchProperties.set(
+      'offset',
+      new gd.PropertyDescriptor(/* defaultValue= */ '100')
+        .setLabel(_('Offset (between -400 and 400)'))
+        .setType('number')
+        .setDescription('The maximum offset amount of slices')
+    );
+    glitchProperties.set(
+      'direction',
+      new gd.PropertyDescriptor(/* defaultValue= */ '0')
+        .setLabel(_('Direction (between -180 and 180)'))
+        .setType('number')
+        .setDescription('The angle in degree of the offset of slices')
+    );
+    glitchProperties.set(
+      'fillMode',
+      new gd.PropertyDescriptor(/* defaultValue= */ '0')
+        .setLabel(_('Fill Mode (between 0 and 4)'))
+        .setType('number')
+        .setDescription(
+          _(
+            'The fill mode of the space after the offset.(0: TRANSPARENT, 1: ORIGINAL, 2: LOOP, 3: CLAMP, 4: MIRROR)'
+          )
+        )
+    );
+    glitchProperties.set(
+      'average',
+      new gd.PropertyDescriptor(/* defaultValue= */ 'false')
+        .setLabel(_('Average'))
+        .setType('boolean')
+        .setDescription(
+          _(
+            'true will divide the bands roughly based on equal amounts where as setting to false will vary the band sizes dramatically (more random looking)'
```

Comment:
This will be shown as a checkbox in the UI, so the description should not contain "true will [...]". It must describe what happen if you check the checkbox :)